src/shared-components/alert/__snapshots__/test.js.snap

```diff
@@ -70,12 +70,12 @@ exports[`Alert UI snapshots renders danger type and text 1`] = `
   display: -webkit-flex;
   display: -ms-flexbox;
   display: flex;
-  padding: 0.25rem 1.5rem;
+  padding: 0 1.5rem 0 1rem;
```

Comment:
@benkolde we still need to clarify this requirement for the sake of Alerts/Tooltip consistency, right now I added a `min-height:48px` to the Alert, It looks the same as Tooltip when there is only 1 line of text but for more lines Alert keeps the original 7px padding and Tooltip 11px

Do you want me to change the padding for Alert to 11px so it is consistent with Tooltip at all times (1 or more lines) and disregard the original requirement of 7px?

Reply:
Alternatively we can tweak Tooltip instead, to match Alert: Add `min-height:48px` and change padding top/bottom to 7px

Reply:
You see: the issue now is the padding for more than 1 line: compare and let me know which one is best for both 7 or 11, or leave them as they are with different padding styles

Reply:
Ah I see what the issue is. 

ALERT: 

Let's go with 11px. That way when it goes to more than one line, the padding stays consistent. The one line height should remain `48px` because  your have `24px (text height) + 22px (padding on top and bottom) + 1px + 1px (for borders) = 48px`